### PR TITLE
relase spring boot 2.3.5

### PR DESCRIPTION
**Pull request by zhoufenqin**
release spring boot starter 2.3.5 version

### CODE REVIEW

**Review by saragluna — APPROVED** [review]
LGTM.

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 2.3.5-beta.1 (Unreleased)
-
+## 2.3.5 (2020-09-14)
+_Bug Fixes_
```
Comment (samvaity) [inline sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:4]: We follow the heading pattern `### Key Bug Fixes` in other changelogs.
Here and other places

File: sdk/spring/azure-spring-boot-starter-active-directory-b2c/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 2.3.5-beta.1 (Unreleased)
+## 2.3.5 (2020-09-14)
```
Comment (samvaity) [inline sdk/spring/azure-spring-boot-starter-active-directory-b2c/CHANGELOG.md:3]: Do we not have any mentions?

Comment (zhoufenqin) [reply sdk/spring/azure-spring-boot-starter-active-directory-b2c/CHANGELOG.md:3]: Hi @samvaity, actually we don't have any feature about `azure-spring-boot-starter-active-directory-b2c` in this version, but we need to unify the azure-spring-boot-starter version.
we have a hotfix before, so some of the spring-boot-starter version is 2.3.4, others are 2.3.3, so in this version, we unify the spring-boot-starter version to 2.3.5

image

**Pull request merged**